src/containers/Login/page.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import Logo from 'components/Auth/Logo';
+import BackgroundLayout from 'components/BackgroundLayout';
+import backgroundImage from 'images/auth_background.png';
+
+import Form from './form';
+
+const Page = () => {
+  const logoLabel = 'Sign in to Nimble';
+  return (
+    <BackgroundLayout backgroundImage={ backgroundImage }>
```

Comment:
The same type of background layout will be used in all the screens and when in the survey list the background image will change dynamically. Making it a div and changing it dynamically works but then all of my screens will have the same divs [whereas this component](https://github.com/rafayet-monon/surveyor/blob/feature/login-screen/src/components/BackgroundLayout/index.js) has it in just one place. So, to avoid code repetition I made it into a component.

Reply:
> And I don't understand about this line https://github.com/rafayet-monon/surveyor/pull/7/files#diff-22fb1272b74d14211e5cdf153b350ec3R3
> Can you tell me more detail about this?

As the background image will be changed dynamically I added it to inline styles. And also there's a fallback color in the design in case there is no background image like this [blank homepage](https://drive.google.com/file/u/1/d/1dC-a1fi40KE-Rzp9w3nRaekX8vnm60HP/view?usp=sharing). I previously added it both in `background` property but now moved the background-color to [styles](https://github.com/rafayet-monon/surveyor/blob/feature/login-screen/src/stylesheets/components/_background-layout.scss#L9)

Reply:
@rafayet-monon But in the design, the login screen has just only 1 background for now.
For the other screen such as the plain color means the other screen.
The screen I that I think we can use the component about the dynamic background is just only the survey screen 🤔

Reply:
Changed in [a21674e](https://github.com/rafayet-monon/surveyor/pull/12/commits/a21674eccfde40c9de1077662052d330f286cdb4)